winpr/libwinpr/sspi/NTLM/ntlm.h

```diff
@@ -79,13 +79,15 @@ typedef enum _NTLM_STATE NTLM_STATE;
 
 enum _NTLM_AV_ID
 {
+#ifndef __MINGW32__
```

Comment:
Sure this breaks nothing? Will change the numerical id of all elements for such builds.